src/main/java/org/cryptomator/ui/mainwindow/MainWindowController.java

```diff
@@ -116,6 +129,34 @@ private void addVault(Path pathToVault) {
 		}
 	}
 
+	private boolean containsUnlockedVaultContent(Path path) {
+		return vaultListManager.list().stream().anyMatch(v -> path.startsWith(v.getAccessPoint()));
+	}
+
+	private Optional<Path> getCiphertextPath(Path path) {
+		return vaultListManager.list().stream() //
```

Comment:
We discussed to define a new drop zone within the unlocked vc. This has two advantages:

1. it is not a hidden feature
2. we don't need to "guess" the vault

Furthermore it doesn't interfere with this existing "add vault" drop zone. Maybe it would be even better to remove this altogether and make only the vault list react to "add vault".

Reply:
Will give it a try.

Reply:
There is one disadvantage though (and that's why I decided for the "guessing" feature): Users have to preselect the correct vault. If they're already trying to drag the encrypted file into Cryptomator and notice that the vault is not selected, they have to cancel the drag, and select the correct vault.